### DISCUSSION TITLE
Creating and destroying groups of resources with Terragrunt

### DISCUSSION THREAD

**Discussion by pete0emerson** [gruntwork-io/knowledge-base#108] [Help] [answered]
We are currently building out our AWS DR infrastructure for our production portals. Our goal is to be able to deploy all infrastructure comprised of many modules with interdependencies using a pipeline in a seamless fashion. Because terragrunt splits state between modules there are challenges deploying a large scale application with short RTO via a pipeline.

Are there any other solutions to creating and destroying groups of resources besides using terragrunt run-all plan/apply?

**Comment by denis256** [comment, marked as answer]
Hi, to deploy/destroy all modules can be used `run-all` command.
